Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM matchoffice/rubies:2.3.1
```

Comment:
I tried both ways - use base elixir and then install rubies, but I got to a point, that installing elixir to rubies is easier.